ecomap/api/v1_0/handlers/comments.py

```diff
@@ -0,0 +1,89 @@
+import tornado.ioloop
+import tornado.web
+from api.v1_0.models.user import User
+from api.v1_0.models.comment import Comment
+from api.v1_0.handlers.base import BaseHandler
+import tornado.escape
+from datetime import datetime
+import json
+
+
+class CommentsHandler(BaseHandler):
+    def get(self, comment_id):
+        json_response = []
+        comments_query = self.sess.query(Comment).filter_by(id=comment_id).all()
+        if not comments_query:
+            self.send_error(status_code=404, message='Not Found')
+        for comment_query in comments_query:
+            response = {
+                'id': comment_query.id,
+                'content': comment_query.content,
+                'problem_id': comment_query.problem_id,
+                'user_id': comment_query.user_id,
+                'datetime': str(comment_query.datetime),
+                'activity_type': comment_query.activity_type
+            }
+            json_response.append(response)
+        self.write(json.dumps(json_response, indent=4))
+
+    def post(self, comment_id):
+        if comment_id:
+            self.send_error(status_code=400, message='Bad Request')
+
+    def put(self, comment_id):
+        comment_query = self.sess.query(Comment).get(comment_id)
+        if not comment_query:
+            self.send_error(status_code=404, message='Not Found')
+        data = tornado.escape.json_decode(self.request.body)
+
+        comment_query.content = data['content']
+        comment_query.datetime = datetime.utcnow()
+        comment_query.user_id = data['user_id']
+        comment_query.activity_type = 'UPDATED'
+        self.sess.commit()
+
+    def delete(self, comment_id):
+        comment_query = self.sess.query(Comment).get(comment_id)
+        if not comment_query:
+            self.send_error(status_code=404, message='Not Found')
+        self.sess.delete(comment_query)
+        self.sess.commit()
+
+
+class ProblemCommentsHandler(BaseHandler):
+    def get(self, problem_id):
+        comments_query = self.sess.query(Comment).filter_by(problem_id=problem_id).all()
+        if not comments_query:
+            self.send_error(status_code=404, message='Not Found')
+
+        json_response = []
+
+        for comment_query in comments_query:
+            user_query = self.sess.query(User).get(comment_query.user_id)
+            response = {
+                'id': comment_query.id,
+                'content': comment_query.content,
+                'first_name': user_query.first_name,
+                'datetime': str(comment_query.datetime),
+                'activity_type': comment_query.activity_type
+            }
+            json_response.append(response)
+        self.write(json.dumps(json_response, indent=4))
+
+    def post(self, problem_id):
+        data = tornado.escape.json_decode(self.request.body)
```

Comment:
we already have this data defiined in the BaseHandler as follows:

json_data = tornado.escape.json_decode(self.request.body)
self.request.arguments.update(json_data)

anyone should use it